test/tf_exec.cpp

```diff
@@ -54,7 +54,6 @@ TEST(tf_exec, hello_world) {
   LOG(INFO) << outputs[0].matrix<float>();
 }
 
-#if !defined(NGRAPH_EMBEDDED_IN_TENSORFLOW)
```

Comment:
Why this change?

Reply:
Not sure how this happened. Changed it back.